.github/workflows/lint.yml

```diff
@@ -0,0 +1,57 @@
+name: lint
+
+on:
+  push:
```

Comment:
should we add here also new GSOC branches?

Reply:
Yes. This is already the case for the python-flake8 workflow on develop, but since this is on stable I missed it. Again, the issue of having two development branches. I'll fix this up.

Reply:
ok on the next merge of develop to stable we have a new major release and also both branches have a similiar test infrastructure.

And yes stable is the branch we do release from. We have a major develop branch and sometimes based on that other feature branches. That is no issue.

Reply:
> ok on the next merge of develop to stable we have a new major release and also both branches have a similiar test infrastructure.

I am not sure if you misunderstood me. I was talking about the next merge from stable to develop, i.e. what should happen right after this PR has been merged, not on the next merge from develop to stable (a major release). These differences between the python-flake8.yml workflow on stable and the python-flake8.yml workflow on develop need to be accounted for in the merge of stable into develop.